src/app/users/user-profile-update/user-profile-update.component.ts

```diff
@@ -0,0 +1,124 @@
+import { Component, OnInit } from '@angular/core';
+import { UserService } from '../../shared/user.service';
+import { Location } from '@angular/common';
+import {
+  FormBuilder,
+  FormControl,
+  FormGroup,
+  FormArray,
+  Validators
+} from '@angular/forms';
+import { CouchService } from '../../shared/couchdb.service';
+import { Observable } from 'rxjs/Observable';
+import { of } from 'rxjs/observable/of';
```

Comment:
I didn;t find the use of this import section over here . Could you please check it once ?

Reply:
i have used it on onsubmit function, Please check...